storage/ast/node.go

```diff
@@ -1,5 +1,5 @@
 /*
-	Copyright 2019 NetFoundry, Inc.
+	Copyright 2020 NetFoundry, Inc.
```

Comment:
Why are we bumping this? I don't believe we need to.

Reply:
Completely agree! It adds tons of noise to PRs, and doesn't really _do_ anything. The copyright is still perfectly in force with, or without this change.

Reply:
I agree, but when I don't do it, @dovholuknf yells at me. Please get @dovholuknf on board, and I'll happily stop :)